File: Web-app-oppgave-2/Models/Lugar.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.Drawing;

namespace Web_app_oppgave_2.Models
{
    public class Lugar
    {
        [Key]
        public int LugarId { get; set; }
        public string Type { get; set; }
        public string Navn { get; set; }
        public int Kapasitet { get; set; }
        public string Beskrivelse { get; set; }
        public byte[] Bilde { get; set; }
        public int MaxReservasjon { get; set; }
        public double Pris { get; set; }
    }
}

File: Web-app-oppgave-2/ClientApp/src/models/lugar.ts
export class Lugar{
  lugarId?:number;
  type:string;
  navn:string;
  kapasistet:number;
  maxReservasjon:number;
  beskrivelse:string;
  bilde?:any;
  pris:number;

  constructor(type:string, navn:string,
    kapasistet:number, maxReservasjon:number,
    beskrivelse:string, pris:number) {
    this.type = type;
    this.navn = navn;
    this.beskrivelse = beskrivelse;
    this.kapasistet = kapasistet;
    this.maxReservasjon = maxReservasjon;
    this.pris = pris;
    //this.bilde = bilde;
  }
}
